app/containers/Header/NotificationIcon.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, memo } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { BG_WARNING_LIGHT, TEXT_DARK } from 'style-constants';
+
+import Span from 'components/Span';
+
+const Div = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 17px;
+  background: ${BG_WARNING_LIGHT};
+  position: ${x => (!x.inline ? 'absolute' : 'relative')};
+  z-index: 10;
+  top: ${x => (!x.inline ? '-6px' : '0')};
+  right: ${x => (!x.inline ? '-6px' : '0')};
+  margin-left: ${x => (x.inline ? '5px' : '0')};
+  border-radius: 8.5px;
+  min-width: 24px;
+
+  $:hover {
+    Tooltip {
+      display: block;
+    }
+  }
+
+  @media only screen and (max-width: 992px) {
+    min-width: 20px;
+  }
+`;
+
+const Base = styled.div`
```

Comment:
`Base` is not a meaningful name for this component. It is used to display a tool tip.